slemicro/combustion/04_install_quemu_guest_agent.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+# Installing qemu-guest-agent service
+if [ "${QEMUGUESTAGENT}" = true ]; then
+    zypper --non-interactive install qemu-guest-agent
```

Comment:
I've tried on an unregistered system and it didn't worked:

```
Warning: No repositories defined. Operating only with the installed resolvables. Nothing can be installed.
Reading installed packages...
Package 'qemu-guest-agent' not found.
Resolving package dependencies...
Nothing to do.
```

I guess you need to make it dependant on the `REGISTER` variable (see https://github.com/suse-edge/misc/blob/main/slemicro/create_vm.sh#L71-L74 for example)